src/tools/cephfs/top/cephfs-top

```diff
@@ -270,22 +292,141 @@ class FSTop(object):
             elif key == curses.KEY_DOWN and curr_row < len(fs_list) - 1:
                 curr_row += 1
             elif (key in [curses.KEY_ENTER, 10, 13]) and fs_list:
-                self.run_display(fs_list[curr_row])
+                sort_dict['last_fs'] = fs_list[curr_row]
+                self.run_display()
                 endmenu = True
             elif key == ord('q'):
-                self.run_all_display()
+                if isinstance(sort_dict['last_fs'], list):
+                    self.run_all_display()
+                else:
+                    self.run_display()
                 endmenu = True
 
             try:
-                self.display_menu(stdscr, curr_row)
+                if not fs_list:
+                    self.display_menu(stdscr)
+                else:
+                    self.display_fs_menu(stdscr, curr_row)
             except curses.error:
                 pass
             curses.halfdelay(self.refresh_interval_secs)
             key = stdscr.getch()
 
+    def choose_field(self, stdscr):
+        curses.curs_set(0)
+        curses.init_pair(1, curses.COLOR_BLACK, curses.COLOR_WHITE)
+        field_menu = ["chit= CAP_HIT", "dlease= DENTRY_LEASE", "ofiles= OPENED_FILES", "oicaps= PINNED_ICAPS",
+                      "oinodes= OPENED_INODES", "rtio= READ_IO_SIZES", "raio= READ_AVG_IO_SIZES",
+                      "rsp= READ_IO_SPEED", "wtio= WRITE_IO_SIZES", "waio= WRITE_AVG_IO_SIZES",
+                      "wsp= WRITE_IO_SPEED", "rlatavg= AVG_READ_LATENCY", "rlatsd= STDEV_READ_LATENCY",
+                      "wlatavg= AVG_WRITE_LATENCY", "wlatsd= STDEV_WRITE_LATENCY", "mlatavg= AVG_METADATA_LATENCY",
+                      "mlatsd= STDEV_METADATA_LATENCY", "Default"]
```

Comment:
Firstly, I tried using that only for easy maintenance but it wasn't successful. Because currently we are using the field names stored in the `field_menu` to match for the field chosen and the display structure of field menu is followed the way it is done in top(1). If I use the `MAIN_WINDOW_TOP_LINE_METRICS` , firstly I will have to call its respective function for eg `self.items()` and self.avg_items()` again and again.Secondly, `MAIN_TOP_LINE_METRICS` have some fields which are not being displayed directly in cephfs-top display as fields, their avg, std is being used but for us, order is very important.That's why, it looked better and easy to do this way.